packages/teams-js/src/public/dialog.ts

```diff
@@ -210,6 +217,121 @@ export namespace dialog {
     }
   }
 
+  /**
+   * @hidden
+   * Hide from docs
+   * --------
+   * Convert AdaptiveCardDialogInfo to DialogInfo to send the information to host in {@linkcode adaptiveCard.open} API.
+   *
+   * @internal
+   */
+  export function getDialogInfoFromAdaptiveCardDialogInfo(adaptiveCardDialogInfo: AdaptiveCardDialogInfo): DialogInfo {
```

Comment:
we avoid exporting internal functions that users don't need.

Reply:
We need to export as we are using them in unit test cases. If we don't export then we cannot test them :)

Reply:
Me and Aaron talked about this (not on your PR, but the same topic). Typically, we would not want to expose some code just for testing purposes. Here is online doc to read more about this. 
**https://enterprisecraftsmanship.com/posts/unit-testing-private-methods/**